src/patterns/check-answers/index.md.njk

```diff
@@ -28,6 +28,14 @@ Check answers pages help to:
 - increase users’ confidence as they can clearly see that they have completed all the sections and that their data has been captured
 - reduce error rates as users are given a second chance to notice and correct errors before submitting data
 
+### Choose the right layout for the page
+
+Many services ask users for short answers, which are suitable for a layout that takes up two-thirds of the screen. For example, the 'Check your answers before submitting your application' page included in this guidance.
```

Comment:
> Many services ask users for short answers, which are suitable for a layout...

Should we link to the [layout guidance](https://design-system.service.gov.uk/styles/layout/) from here ^?

> ...that takes up two-thirds of the screen.

Does this ^ only apply to desktop devices?

Reply:
```suggestion
Many services ask users for short answers, which are suitable for a layout that takes up two-thirds of the screen on larger devices. For example, the 'Check your answers before submitting your application' page included in this guidance.
```

Reply:
@calvin-lau-sig7 Nice!

Do you think we should link to the Layout guidance?